software/FossaSat2/src/ADCS/ADS/kalman_filtering.cpp

```diff
@@ -12,56 +12,70 @@
 
 
 /*************** Auxiliary functions implementation *****************/
-void ADS_Inverse_Matrix(ADCS_CALC_TYPE matrix[ADCS_STATE_DIM][ADCS_STATE_DIM]) {
+ADCS_CALC_TYPE absolute(ADCS_CALC_TYPE value)
+{
+    ADCS_CALC_TYPE abs_value;
+
+    // Take the absolute value of the parameter
+    if (value < 0)
+        abs_value = -1.0*value;
+
+    return (value);
+}
```

Comment:
This seems redundant, there's alrady abs() macro. Also it always returns `value`.